static/js/recent_senders.js

```diff
@@ -61,4 +118,23 @@ exports.compare_by_recency = function (user_a, user_b, stream_id, topic) {
     return 0;
 };
 
+exports.get_topic_recent_senders = function (stream_id, topic) {
+    const topic_dict = topic_senders.get(stream_id);
+    if (topic_dict === undefined) {
+        return [];
+    }
+
+    const sender_message_ids = topic_dict.get(topic);
+    if (sender_message_ids === undefined) {
+        return [];
+    }
+
+    const sorted_senders = Array.from(sender_message_ids.entries()).sort(
+        (s1, s2) => { return s1[1] - s2[1]; }
+    );
+    const recent_senders = [];
+    sorted_senders.forEach((item) => { recent_senders.push(item[0]); });
```

Comment:
We've generally been moving loops to use `for (const item of sorted_senders) {` as our default loop style; can you make this PR use that as well?  `.forEach` has various issues.

Reply:
@andersk do we want to just bulk-migrate and link against `.forEach`?

Reply:
Interesting! Thanks for noticing this.